integration/config/etc/containerd/config.toml

```diff
@@ -1,6 +1,16 @@
 # explicitly use v2 config format
 version = 2
 
+disabled_plugins = [
+	"io.containerd.snapshotter.v1.aufs",
+	"io.containerd.snapshotter.v1.btrfs",
+	"io.containerd.snapshotter.v1.devmapper",
+	"io.containerd.snapshotter.v1.zfs",
+	"io.containerd.tracing.processor.v1.otlp",
+	"io.containerd.internal.v1.tracing",
+	"io.containerd.grpc.v1.cri",
```

Comment:
We currently successfully load the CRI plugin with a valid config. Is there any useful insight to share here before disabling it, even if tests are still passing?

Reply:
From #377 
```
{"error":"cni config load failed: no network config found in /etc/cni/net.d: cni plugin not initialized: failed to load cni config","level":"error","msg":"failed to load cni during init, please check CRI plugin status before setting up network for pods"
```

This is coming from the CRI plugin.

Reply:
Yes, but this says cni failed to load because there is no network config. cri is loading fine, I think? I guess we don't need it, since the tests pass, but disabling it isn't the same as disabling the ones that fail to load.

Reply:
"cni plugin" here is not referring to a containerd plugin. CNI itself has a plugin system that allows you to write code that can set up arbitrary network configurations, you then string those plugins together via a network config. By default, the containerd CRI plugin assumes there will be a CNI network config of CNI plugins at `/etc/cni/net.d` and will try to load it. If it can't, CRI still starts up, but at this point it can't set up networking so it logs the error.

We could alternatively configure CRI to not look for that default CNI config (I think. It might override an empty config with the default that logs that message). Since we're not testing against kubernetes right now, I thought disabling the plugin would be easier than configuring it in a way that won't work if/when we do want to test against kubernetes.